docs/concepts/clickhouse.rst

```diff
@@ -0,0 +1,24 @@
+Clickhouse
+##################################
+
+ClickHouse is a high-performance, column-oriented SQL database management system (DBMS) 
+for online analytical processing (OLAP). It is used as the backend storage for the Aspects 
+project that powers superset.
+
```

Comment:
Can you add a section here about how ClickHouse can be run locally, or using ClickHouse Cloud (or other cloud service providers)?